doc/source/learning/overview.rst

```diff
@@ -0,0 +1,38 @@
+Overview
+=====================
+
+This part of the documentation will help new developers to understand what is Airship and all its
+ecosystem of tools.
+
+This provides links to documentation and videos to help learning, but also to as an entry point to
+find the important documentation.
+
+First let's see how the Airship parts are going to work together.
+
+
+What is Airship?
++++++++++++++++++
```

Comment:
I would remove this section.
This is not necessary for socok8s, as we are not leveraging drydock afaik.
IMO all the airship features and links should be in the airship part of the documentation, above.

Here we should explain how this repo does things at a high level:
- Use run.sh to run ansible scripts
- Explain what those ansible scripts do
- Explain how terraform would be leveraged
- ... (Explain the rest of the flow)

Each of those sections can be pointing to the docs. The idea here is to explain _how to develop_ on those parts. So explaining their duty and how to modify them sounds very nice.

Reply:
Removed. I moved install Caasp 3 to the overview. And I am going to add this information you mentioned.